src/gurt/telemetry.c

```diff
@@ -2342,15 +2339,14 @@ uint64_t *
 d_tm_allocate_shared_memory(int srv_idx, size_t mem_size)
 {
 	key_t	key;
-	int	shmid;
```

Comment:
Not sure I fully understand why this is now a global. Is there some reason the shmid shouldn't be scoped to this function?

Reply:
The shmid is required for the shmctl which uses it to identify the segment it should remove.  This wasn't needed before, so the value was kept local.